quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/KeycloakRecorder.java

```diff
@@ -68,6 +70,21 @@ public void configureProfile(Profile.ProfileName profileName, Map<Profile.Featur
         Profile.init(profileName, features);
     }
 
+    public void configureTruststore() {
+        String[] truststores = Configuration.getOptionalKcValue(SecurityOptions.TRUSTSTORE_PATHS.getKey())
+                .map(s -> s.split(",")).orElse(null);
+
+        String dataDir = Environment.getDataDir();
+
+        File truststoresDir = Environment.getHomePath().resolve("conf").resolve("truststores").toFile();
```

Comment:
Should we create that dir OOTB. Not by code here, I mean. It should be present in the distribution prior to even starting the server for the first time.

Reply:
Added the empty dir creation to the assembly.